Knowledge/react-app/src/components/Searchbar.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import Grid from './Grid';
+import '../css/searchbar.scss';
+import '../css/grid.scss';
+
+const URL_TO_RETRIEVE_DATA = 'https://api.myjson.com/bins/uptto';
+var constData = {};
+
+class Searchbar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentdata: {},
+      userInput: ''
+    };
+
+    this.updateInputValue = this.updateInputValue.bind(this);
```

Comment:
you can delete this and define `this.updateInputValue` as shown in below comment